app/assets/stylesheets/_base.scss

```diff
@@ -6,3 +6,13 @@ html{
 body{
   padding: 100px 20px 100px 20px;
 }
+
+hr{
+  border-top: 1px solid #D4D4D4;
```

Comment:
Color literals like `#D4D4D4` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#D4D4D4` should be written as `#d4d4d4`